bookstore/publish.py

```diff
@@ -1,16 +1,12 @@
 import json
 
 import aiobotocore
-
-from notebook.base.handlers import APIHandler
-from notebook.base.handlers import path_regex
+from notebook.base.handlers import APIHandler, path_regex
 from notebook.utils import url_path_join
 from tornado import web
 
-from .s3_paths import s3_path
-from .s3_paths import s3_key
-from .s3_paths import s3_display_path
 from .bookstore_config import BookstoreSettings
+from .s3_paths import s3_display_path, s3_key, s3_path
```

Comment:
I had split these out because of your previous styling edit that took my one line imports and split them into multiple. I prefer them to be on one line, but was trying to incorporate your changes.

Can you give me guidance on when to go in one direction or the other?

Reply:
I tend to let the linter do whatever the linter likes unless it's really jarring. I generally leave all import from the same path in the same statement and let it spill to multi-line over separate repeated imports. Otherwise it's easy to miss duplicate imports and adds confusion around what importing is doing.Though that's kjust my opinion on the topic.

Reply:
I agree, I prefer to let the linter decide, but I was trying to incorporate the changes @willingc  had made in this PR: https://github.com/nteract/bookstore/pull/70/files#diff-b79c22e74e3095962bbd282b0bd0c040R13